### PR TITLE
Move implementers page to this repo

### PR DESCRIPTION
**Pull request by hasheddan**
Adds page for projects or companies currently engaged in efforts to
support OCI artifacts.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

### CODE REVIEW

**Comment by hasheddan** [comment]
@mikebrow sounds good! Done :heavy_check_mark:

**Comment by jzelinskie** [comment]
Files like this get stale quick and there's no real way to validate this list.
Can we instead provide something that registries can put on their website/readme?

**Comment by hasheddan** [comment]
@jzelinskie SGTM, I would recommend removing this link as was the original proposal, what do you think?

**Comment by mikebrow** [comment]
I'm fine with duping this one to https://github.com/opencontainers/artifacts/pull/4 or approving this edit for now and rebasing #4.

> Files like this get stale quick and there's no real way to validate this list.

Intent is to have test cases for validation. As Steve wrote, these are but a couple wip projects being used for a proof of concept. 

> Can we instead provide something that registries can put on their website/readme?

We can also provide something that registries can put on their website (wip). This PR is just moving the current link from a private repo to this repo.

**Comment by josephschorr** [comment]
@mikebrow Perhaps we get a test suite added to the OCI Compliance suite behind one of its testing options and just reference that and allow registries to (re)use the same OCI compliance badging?

**Comment by mikebrow** [comment]
> @mikebrow Perhaps we get a test suite added to the OCI Compliance suite behind one of its testing options and just reference that and allow registries to (re)use the same OCI compliance badging?

absolutely.. just need to get along with that suite.. the dist. spec.. and this guidance repo.. 

In the mean time :-) .. we have a file that has a link to a remote repo. We should discuss the contents in #4 where steve holds the pen.. This pr is just fixing the link.. for the most part (IMO) not the contents of what it's linking to (#4).

**Comment by hasheddan** [comment]
@mikebrow done :heavy_check_mark:

**Comment by SteveLasker** [comment]
@hasheddan, has Docker Hub completed the update to support expanded `mediaTypes`?
I realized this PR is referencing some stale content in the [definitions-terms.md](https://github.com/opencontainers/artifacts/pull/24/files#diff-da55140985df1701b1b0ece67f61bfde0bc2e2e85aeea1cb3d9bba978945b1de) file. I'd like to merge and fix, but wanted to confirm Docker completed the work. Others were asking for an implementors PR as well.

**Comment by hasheddan** [comment]
@SteveLasker it appears the work is still in progress (https://github.com/distribution/distribution/pull/2934), though it does look pretty stale at this point. Not sure if this would fall into the category of still "engaging in efforts", but I presume it is still desired, especially with recent CNCF news.

/cc @jdolitsky

**Comment by jdolitsky** [comment]
Docker Hub is not mentioned in this PR. Distribution (the opensource project) has supported the artifacts stuff since the very beginning, running with default settings.

**Comment by hasheddan** [comment]
@jdolitsky thanks for the clarification. @SteveLasker I assumed you were talking about distribution since that is what is referenced in `implementers.md` here, but do we need to ensure that docker hub also supports?

**Comment by SteveLasker** [comment]
woops, I read docker/distribution as docker hub. Too much multi-tasking.
@hasheddan, if you can just change to [distribution/distribution](https://github.com/distribution/distribution), we can go ahead and merge.

**Comment by hasheddan** [comment]
@SteveLasker done :+1:

File: implementers.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# OCI Artifacts Implementations
+
+Projects or companies currently engaged in efforts to support OCI Artifacts
+
+* [Azure Container Registry](https://aka.ms/acr/artifacts)
+* [distribution/distribution](https://github.com/distribution/distribution/) - the reference implementation of the [OCI Distribution Spec][distribution-spec]
```
Comment (jdolitsky) [inline implementers.md:6]: we should drop "the reference implementation"

**Review by jdolitsky — CHANGES_REQUESTED** [review]

File: implementers.md (hunk 1 above)
Comment (hasheddan) [reply implementers.md]: @jdolitsky done :+1:

**Review by jdolitsky — APPROVED** [review]

**Comment by SteveLasker** [comment]
LGTM

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by mikebrow — APPROVED** [review]
LGTM

**Pull request merged**